scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1403,6 +1418,29 @@ def fetch_fid(self, service, idx):
     fid = get_fid(self, fids, service, idx)
     return fid
 
+def getListOfm0dProcess():
+    '''
+    Get list of running m0d process
+    '''
+    listOfProc = []
+    # Iterate over the list
+    for proc in psutil.process_iter():
+       try:
+           # Fetch process details as dict
+           pinfo = proc.as_dict(attrs=['pid', 'name', 'username'])
+           if pinfo.get('name') == "m0d":
```

Comment:
Should the name == "m0d" or "lt-m0d" ?

Reply:
`[root@cortx-data-headless-svc-ssc-vm-rhev4-2078 /]# ps aux | grep m0d
root         99  0.0  0.0  12268  1656 ?        S    05:36   0:00 /usr/bin/bash /usr/libexec/cortx-motr/motr-start m0d-0x7200000000000001:0xa
root        100  0.2  1.9 14786364 318792 ?     Sl   05:36   0:40 /usr/bin/m0d -e libfab:inet:tcp:cortx-data-headless-svc-ssc-vm-rhev4-2078@21001 -A linuxstob:/etc/cortx/log/motr/0a75d9db67d245bab0ee37d614317067/addb/m0d-0x7200000000000001:0xa/addb-stobs -f <0x7200000000000001:0xa> -T ad -S stobs -D db -m 524288 -q 16 -H inet:tcp:cortx-data-headless-svc-ssc-vm-rhev4-2078@22001 -U -B /dev/sdg1 -z 9999220736 -r 134217728
`
The process name running in containers is m0d. It looks good.